models/pathology_nuclei_segmentation_classification/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_hovernet_20221124.json",
-    "version": "0.1.7",
+    "version": "0.1.8",
     "changelog": {
+        "0.1.8": "modify readme",
```

Comment:
Please write more info, for example: Update README for XXX.

Reply:
Have updated, thanks!